prometheus/prometheus.spec

```diff
@@ -0,0 +1,205 @@
+results, and can trigger alerts if some condition is observed to be true.
+
+################################################################################
+
+%prep
+%setup -qn %{name}-%{version}
+
+%build
+export GOPATH=$(pwd)
+go get github.com/%{name}/%{name}/cmd/...
+
+%{__make} %{?_smp_mflags} build
+
+%install
+rm -rf %{buildroot}
+
+install -dm 0755 %{buildroot}%{_bindir}
+install -dm 0755 %{buildroot}%{_sysconfdir}/%{name}
+install -dm 0755 %{buildroot}%{_sysconfdir}/sysconfig
+install -dm 0755 %{buildroot}%{_datarootdir}/%{name}/consoles
+install -dm 0755 %{buildroot}%{_datarootdir}/%{name}/console_libraries
+install -dm 0755 %{buildroot}%{_sharedstatedir}/%{name}
+install -dm 0755 %{buildroot}%{_logdir}/%{name}
+
+%if 0%{?rhel} >= 7
+    install -dm 0755 %{buildroot}%{_unitdir}
+%else
+    install -dm 0755 %{buildroot}%{_initrddir}
+%endif
+
+install -pm 755 prometheus %{buildroot}%{_bindir}/%{name}
+install -pm 755 promtool %{buildroot}%{_bindir}/promtool
+
+for dir in consoles console_libraries; do
+  for file in ${dir}/*; do
+    install -pm 0644 ${file} %{buildroot}%{_datarootdir}/%{name}/${file}
+  done
+done
+
+install -pm 0644 documentation/examples/%{name}.yml %{buildroot}%{_sysconfdir}/%{name}/%{name}.yml
+install -pm 0644 %{SOURCE3} %{buildroot}%{_sysconfdir}/sysconfig/%{name}
+install -pm 0644 %{SOURCE4} %{buildroot}%{_sysconfdir}/logrotate.d/%{name}
+
+%if 0%{?rhel} >= 7
+    install -pm 0644 %{SOURCE1} %{buildroot}%{_unitdir}/%{name}.service
+%else
+    install -pm 0755 %{SOURCE2} %{buildroot}%{_initrddir}/%{name}
+%endif
+
+%clean
+rm -rf %{buildroot}
+
+################################################################################
+
+%pre
+%{__getent} group %{service_group} >/dev/null || %{__groupadd} -r %{service_group}
+%{__getent} passwd %{service_user} >/dev/null || %{__useradd} -r \
+    -g %{service_group} -d %{_sharedstatedir}/%{name} \
+    -s /sbin/nologin %{service_user}
+
+%post
+if [[ $1 -eq 1 ]] ; then
+%if 0%{?rhel} >= 7
+  %{__systemctl} enable %{name}.service &>/dev/null || :
+%else
+  %{__chkconfig} --add %{name} &>/dev/null || :
+%endif
+fi
+
+%preun
+if [[ $1 -eq 0 ]] ; then
+%if 0%{?rhel} >= 7
+  %{__systemctl} --no-reload disable %{name}.service &>/dev/null || :
+  %{__systemctl} stop %{name}.service &>/dev/null || :
+%else
+  %{__service} %{name} stop &>/dev/null || :
+  %{__chkconfig} --del %{name} &>/dev/null || :
+%endif
+fi
+
+%postun
+%if 0%{?rhel} >= 7
+if [[ $1 -ge 1 ]] ; then
+  %{__systemctl} daemon-reload &>/dev/null || :
+fi
+%endif
+
+################################################################################
+
+%files
+%defattr(-,root,root,-)
+%{_bindir}/%{name}
+%{_bindir}/promtool
+%config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
+%config(noreplace) %{_sysconfdir}/%{name}/%{name}.yml
+%{_datarootdir}/%{name}
+%if 0%{?rhel} >= 7
+    %{_unitdir}/%{name}.service
+%else
+    %{_initrddir}/%{name}
```

Comment:
Don't use prefix spaces in `%files` section.

Reply:
Fixed (b3cacf056485e9b0e5baa5acf1caef634e2ac020).